assets/xml/objects/object_po_composer.xml

```diff
@@ -1,35 +1,36 @@
 ﻿<Root>
+    <ExternalFile XmlPath="objects/gameplay_keep.xml" OutPath="assets/objects/gameplay_keep/"/>
+
     <File Name="object_po_composer" Segment="6">
-        <Animation Name="object_po_composer_Anim_00020C" Offset="0x20C" />
-        <Animation Name="object_po_composer_Anim_000570" Offset="0x570" />
-        <Animation Name="object_po_composer_Anim_000A58" Offset="0xA58" />
-        <Animation Name="object_po_composer_Anim_001420" Offset="0x1420" />
-        <Animation Name="object_po_composer_Anim_0015B8" Offset="0x15B8" />
-        <Animation Name="object_po_composer_Anim_00188C" Offset="0x188C" />
-        <Animation Name="object_po_composer_Anim_001F44" Offset="0x1F44" />
-        <Animation Name="object_po_composer_Anim_0023E8" Offset="0x23E8" />
-        <Animation Name="object_po_composer_Anim_002AE0" Offset="0x2AE0" />
-        <Animation Name="object_po_composer_Anim_003024" Offset="0x3024" />
-        <Animation Name="object_po_composer_Anim_00334C" Offset="0x334C" />
-        <Animation Name="object_po_composer_Anim_003954" Offset="0x3954" />
-        <Animation Name="object_po_composer_Anim_003DB0" Offset="0x3DB0" />
-        <Texture Name="object_po_composer_Tex_003DC0" OutName="tex_003DC0" Format="i8" Width="32" Height="64" Offset="0x3DC0" />
+        <!-- Animations -->
+        <Animation Name="gPoeComposerAttackAnim" Offset="0x20C" />
+        <Animation Name="gPoeComposerRaiseArmsAnim" Offset="0x570" />
+        <Animation Name="gPoeComposerBeginPlayingAnim" Offset="0xA58" />
+        <Animation Name="gPoeComposerPlayingAnim" Offset="0x1420" />
+        <Animation Name="gPoeComposerFleeAnim" Offset="0x15B8" />
+        <Animation Name="gPoeComposerIdleAnim" Offset="0x188C" />
+        <Animation Name="gPoeComposerArmsRaisedAnim" Offset="0x1F44" />
+        <Animation Name="gPoeComposerLowerArmsAnim" Offset="0x23E8" />
+        <Animation Name="gPoeComposerIdleLookingDownAnim" Offset="0x2AE0" />
+        <Animation Name="gPoeComposerBeginRollingAnim" Offset="0x3024" />
+        <Animation Name="gPoeComposerRollingAnim" Offset="0x334C" />
+        <Animation Name="gPoeComposerAppearDisappearAnim" Offset="0x3954" />
+        <Animation Name="gPoeComposerIdleStillAnim" Offset="0x3DB0" />
```

Comment:
Hmm, I see why you've named it like this (because you already used "Idle" for `gPoeComposerIdleLookingDownAnim`), but maybe we can just drop "Idle" from that one and just call this one `gPoeComposerIdleStillAnim`. What do you think?